odl/contrib/param_opt/param_opt.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Method(s) to find optimal reconstruction/regularization parameters with
+respect to given figures of merit."""
+
+import numpy as np
+import scipy
+
+__all__ = ('optimal_parameters', )
+
+
+def optimal_parameters(reconstruction, fom, phantoms, data,
+                       initial_param=0):
+    """Find the optimal parameters for a reconstruction method.
+
+    Notes
+    -----
+    For a forward operator :math:`A : X \\to Y`, a reconstruction operator
```

Comment:
Prefer to change this docstring to use  `r"""` and then remove the double backslashes